site/en/install/gpu.md

```diff
@@ -109,10 +109,6 @@ wget http://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1
 <code class="devsite-terminal">sudo apt install ./nvidia-machine-learning-repo-ubuntu1804_1.0.0-1_amd64.deb</code>
 <code class="devsite-terminal">sudo apt-get update</code>
 
-# Install NVIDIA driver
-<code class="devsite-terminal">sudo apt-get install --no-install-recommends nvidia-driver-450</code>
-# Reboot. Check that GPUs are visible using the command: nvidia-smi
```

Comment:
Should we still keep this line somehow, about verifying that the GPUs are visible?

Reply:
@sanjoy Agreed. Running `nvidia-smi` is a good practice 👍 

Suggestion:

```
# Check that GPUs are visible using the command:
<code class="devsite-terminal">nvidia-smi</code>
```

(This way the user would be more inclined to run the step instead of having an option to run it IMO)

cc @lamberta

Reply:
@sanjoy,
Done. Added the line to check GPU visibility, after the drivers are installed. 

Thanks!